### PR TITLE
chore: render harcoded promo codes for local env

### PR DESCRIPTION
**Pull request by leoslopez**
image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -19,6 +19,51 @@
...
+                    type: 'percen',
+                    value: 15,
+                    useLimit: 1,
+                    minPaymentAmount: 1,
+                  },
+                ],
+              },
+              { targetOrigin: '*' },
+            );
+          },
```
Comment (andresmoschini) [inline public/index.html:64]: What do you think about moving this code to [App.tsx](https://github.com/FromDoppler/unlayer-editor/blob/main/src/App.tsx)?

Comment (andresmoschini) [reply public/index.html:64]: > let me see how to add it in the "React way".

Oh, you are right, in that case, probably adding it to [index.tsx](https://github.com/FromDoppler/unlayer-editor/blob/main/src/index.tsx) as plain javascript could be a better option

Comment (andresmoschini) [reply public/index.html:64]: In Editors WebApp, we are not using the "React way": https://github.com/FromDoppler/doppler-editors-webapp/pull/661/files#diff-0b5adbfe7b36e4ae2f479291e20152e33e940f7f265162d77f40f6bdb5da7405R23-R24

Comment (leoslopez) [reply public/index.html]: done.

File: public/index.html
Comment (andresmoschini) [inline public/index.html:1]: Remember to revert these changes

File: src/index.tsx [hunk 2]
```diff
@@ -2,9 +2,12 @@ import React, { StrictMode } from 'react';
 import { createRoot } from 'react-dom/client';
 import './index.css';
 import App from './App';
+import { registerListeners } from './editorsWebappListenersRegistrator';
 
 const container = document.getElementById('unlayer-editor-container');
 
+registerListeners();
```
Comment (andresmoschini) [inline src/index.tsx:9]: ❤️
